notebooks/Composer_Functional.ipynb

```diff
@@ -200,7 +203,8 @@
     "model = Net()\n",
     "# only use one data augmentation since our small model runs quickly\n",
     "# and allows the dataloader little time to do anything fancy\n",
-    "train_and_eval(model, trainloader, testloader)"
+    "device = 'cuda' if torch.cuda.is_available() else 'cpu'\n",
```

Comment:
Can we rm the "device" arg in `train_and_eval` and set it using this line (within the function), instead of duplicating this line each time we call `train_and_eval`?

Reply:
Done